Controllers/BlogController.cs

```diff
@@ -25,7 +25,7 @@ public async Task<ActionResult<IEnumerable<Blog>>> GetBlogs()
         return Ok(_mapper.Map<IEnumerable<Blog>>(blogEntities));
     }
 
-    [HttpGet("{id}")]
+    [HttpGet("{id}", Name = "GetBlog")]
```

Comment:
Typically when working with swagger you'll want to have attributes to tell swagger how it should generate your documentation. It also allows you to show the types that are being returned. Here are examples of those attributes that go on a method.

[ProducesResponseType(StatusCodes.Status200OK)]
[ProducesResponseType(typeof(ExampleResponseObject), StatusCodes.Status400BadRequest)]
[ProducesResponseType(typeof(ExampleResponseObject), StatusCodes.Status404NotFound)]